File: leetcode/move-zeroes.cpp
#include <bits/stdc++.h>
#include <unordered_map>
#include <string>
#include <cctype>
#include <vector>
#include <algorithm>
#include <iostream>
#include <variant>
#include <any>
#include <ctype.h>
#include <stdio.h>
using namespace std;
class Solution {
public:
void moveZeroes(vector<int> &nums) {
    int c = 0;
    bool cond = true;
    if (find(nums.begin(), nums.end(), 0) == nums.end()) {
        cond = false;}

    if (cond == true) {
    for (int i = 0; i < nums.size(); ++i) {
        if (nums[i] != 0) {
            nums[c] = nums[i];
            c++;}
    }
    for (int z = c; z < nums.size(); z++) {
        nums[z] = 0;
    }
    }



}
};

File: leetcode/move-zeroes.py
class Solution:
    def moveZeroes(self, nums: list[int]) -> None:
        """
        Do not return anything, modify nums in-place instead.
        """

        count = 0
        for x in range(len(nums)):
            if nums[x] != 0:
                nums[count] = nums[x]
                count += 1
        for x in range(count, len(nums)):
            nums[x] = 0
        return nums

